docs/open-source.mdx

```diff
@@ -4,51 +4,54 @@ title: Ory is open source
 
-## GitHub
+## Projects
 
-You can find our open source projects on GitHub:
+You can find all of our open source projects on GitHub:
 
-**Fully-featured servers**
+### Fully-featured server
 
 - [Ory Kratos Identity Server](https://github.com/ory/kratos)
 - [Ory Hydra Federation Server](https://github.com/ory/hydra)
 - [Ory Oathkeeper](https://github.com/ory/oathkeeper)
 - [Ory Keto Permission Server](https://github.com/ory/keto)
 
-**Libraries**
+### Library
 
+- [Ory SDK](https://github.com/ory/sdk)
+  - Software development kit for all popular programming languages, generated using openapi-generator.
+- [Ory Dockertest](https://github.com/ory/dockertest)
+  - Dockertest helps you boot up docker images for your Go tests with minimal work.
+    [Ory Dockertest Documentation](https://godoc.org/github.com/ory/dockertest) .
+- [Ory Kubernetes Helm Charts](https://github.com/ory/k8s)
+  - For a list of Helm charts and their configuration options please visit [the k8s documentation](https://k8s.ory.sh/helm/).
 - [Ory Fosite](https://github.com/ory/fosite)
+  - Extensible OAuth 2.0 and OpenID Connect SDK for Go. [Ory Fosite Documentation](https://godoc.org/github.com/ory/fosite).
 - [Ory Ladon](https://github.com/ory/ladon)
-- [Ory Dockertest](https://github.com/ory/dockertest)
-
-## Documentation
-
-- [Identity and sessions (Ory Kratos)](https://www.ory.sh/docs/kratos/)
-- [OAuth 2.0 & OpenIDConnect (Ory Hydra)](https://www.ory.sh/docs/hydra/5min-tutorial)
-- [Permissions & relationships (Ory Keto)](https://www.ory.sh/docs/keto/concepts/relation-tuples/)
-- [Identity Aware Proxy / Zero Trust Networking (Ory Oathkeeper)](https://www.ory.sh/docs/oathkeeper/)
-
-REST APIs are documented using swagger and can be viewed here:
-
-- [Ory Kratos API](https://www.ory.sh/docs/kratos/reference/api/)
-- [Ory Hydra API](https://www.ory.sh/docs/hydra/reference/api/)
-- [Ory Keto API](https://www.ory.sh/docs/keto/reference/api/)
-- [Ory Oathkeeper API](https://www.ory.sh/docs/oathkeeper/reference/api/)
-
-## Library documentation
-
-GoDoc documents Go code and is available for our Go libraries:
-
-- [Ory Fosite](https://godoc.org/github.com/ory/fosite)
-- [Ory Dockertest](https://godoc.org/github.com/ory/dockertest)
-- [Ory Ladon](https://godoc.org/github.com/ory/ladon)
+  - An SDK for access control policies: authorization for the microservice and IoT age. Inspired by AWS IAM policies.
+    [Ory Ladon Documentation](https://godoc.org/github.com/ory/ladon).
+- [Ory Herodot](https://github.com/ory/herodot)
+  - A lightweight Go library for writing responses and errors to HTTP, serves millions of requests daily through Ory Hydra.
+- [Ory Graceful](https://github.com/ory/graceful)
+  - Best practice HTTP server configurations and helpers for Go 1.8's HTTP graceful shutdown feature.
+- [Ory go-acc](https://github.com/ory/go-acc)
+  - A tool for reporting accurate Code Coverage in Golang. Cross platform (OsX, Windows, Linux).
```

Comment:
How about putting this in a table? I think it'll look better than this two-tier list.

Reply:
How should that look like in your opinion? 

like this maybe? 

| Project   | Description | Documentation |
|-----------|-------------|  ---- |
| [Ory Ladon](https://github.com/ory/ladon) | SDK for access control policies: authorization for the microservice and IoT age. Inspired by AWS IAM policies. | some link
| [Ory Herodot](https://github.com/ory/herodot) | A lightweight Go library for writing responses and errors to HTTP, serves millions of requests daily through Ory Hydra. | some link |